JetHerald/Commands/DiscordCommands.cs

```diff
@@ -0,0 +1,116 @@
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using MySql.Data.MySqlClient;
+
+namespace JetHerald
+{
+    [ModuleLifespan(ModuleLifespan.Transient)]
+    public class DiscordCommands : BaseCommandModule
+    {
+        public Db db { private get; set; }
+
+        [Command("createtopic")]
+        [Description("Creates a topic.")]
+        [RequireDirectMessage]
+        public async Task CreateTopic(
+            CommandContext ctx,
+            [Description("The unique name of the new topic.")]
+            string name,
+            [RemainingText, Description("The name displayed in service messages. Defaults to `name`")]
+            string description = null)
+        {
+            if (description == null)
+                description = name;
+
+            await ctx.TriggerTypingAsync();
```

Comment:
Try removing `await` here and see if it works.
Same for all other places where this is sent.